FourSquare/Define/SlideMenu.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  ChangeRootSlideMenu.swift
+//  FourSquare
+//
+//  Created by Mylo Ho on 8/2/16.
+//  Copyright © 2016 Le Van Binh. All rights reserved.
+//
+
+import UIKit
+
+class SlideMenu {
+    static let getRootBackground = (UIApplication.sharedApplication().delegate?.window!!.rootViewController as? BackgroundViewController)!
```

Comment:
Please do not use implicitly unwrapped optional
Do not make define for this. Make an extension